frontends/web/src/routes/account/info/signingconfiguration.tsx

```diff
@@ -59,6 +59,19 @@ class SigningConfiguration extends Component<Props, State> {
         apiPost(`account/${this.props.code}/verify-extended-public-key`, index);
     }
 
+    private scriptTypeTitle = (scriptType: string) => {
+        switch (scriptType) {
+            case 'p2pkh':
+                return 'Legacy';
+            case 'p2wpkh-p2sh':
+                return 'Segwit';
+            case 'p2wpkh':
+                return 'Native segwit';
```

Comment:
`SegWit`? Also, not mentioning Bech32?

Reply:
We use `Title case`, not every word capitalized, everywhere.

Reply:
> Also, not mentioning Bech32?

The xpub has nothing to do with bech32.

Reply:
using `Native segwit (bech32)` now :shrug:

Reply:
> We use Title case, not every word capitalized, everywhere.

But it's a single word, an abbreviation. By this logic, it should be `Bitboxapp` then. Or is that an exception? :)

Reply:
Ah I thought you meant the whole Native **S**egwit. Anyway, as mentioned in the other comment, we use `Segwit` everywhere in the guides as well.